.travis.yml

```diff
@@ -1,23 +1,11 @@
 language: bash
-dist: xenial
```

Comment:
xenial is default according to https://docs.travis-ci.com/user/reference/overview/

Changing the default image does not seem like something Travis should do recklessly. 
Nice :+1: